README.md

```diff
@@ -37,9 +33,11 @@ use bevy_web_asset::WebAssetPlugin;
 
 fn main() {
     App::new()
-        // The `WebAssetPlugin` must be inserted instead of `AssetPlugin` and before `DefaultPlugins`
-        .add_plugin(WebAssetPlugin::default())
-        .add_plugins(DefaultPlugins.build().disable::<AssetPlugin>())
+        // The `WebAssetPlugin` must be inserted before the `AssetPlugin`
+        .add_plugins((
+            WebAssetPlugin::default(),
```

Comment:
I take it you tested the other way around?

Reply:
It just repeats the documentation of `App::register_asset_source`:

```
Registers the given AssetSourceBuilder with the given id.

Note that asset sources must be registered before adding AssetPlugin to your application, since registered asset sources are built at that point and not after.
```